docs/book/deploying-zenml/zenml-cloud/cloud-system-architecture.md

```diff
@@ -0,0 +1,98 @@
+---
+description: Different variations of the ZenML Cloud architecture depending on your needs.
+---
+
+# ☁ Cloud Architecture
```

Comment:
What is this emoji? Typo?

Reply:
ah yes I see it now. It looked like a camel against the dark background somehow. Anyway seems fine.